### ISSUE TITLE
Usage of Personal "Du"-Addressing in Test Registration Process

### ISSUE THREAD

**Issue by jkrwdf** [corona-warn-app/cwa-documentation#831]
The CWA generally uses the German "Sie" form to address the user. 

Examples: 
"Sie lassen sich testen?"
"Finden Sie Teststellen..."
"Ihre Zertifikate erfüllen die..."

However, in this dialog (appears when registering a test) the "Du"-form is used:

![image](https://user-images.githubusercontent.com/5329491/151751961-1872145e-475f-4499-bc96-97fd05d3d0d3.png)

**Comment by Ein-Tim** [comment]
This pop up comes from Android and is not controlled by the CWA AFAIK.

**Comment by MikeMcC399** [comment]
@jkrwdf
This is from an Android device, correct?

If you search through the Android sources on https://github.com/corona-warn-app/cwa-app-android/tree/release/2.16.x you will not find the text which you quoted. Google is referring to the "Gesundheitsbehörde" which is represented in this case by the Robert Koch Institute as the publisher and operator of the Corona-Warn-App and backend systems. If the text were to be coming from the app RKI wouldn't refer to itself in that way.

Google uses du, whereas CWA uses Sie. You can see this also if you select
Einstellungen > Google > COVID-19-Benachrichtigungen

The text there from Google also uses "du" and refers to the "Gesundheitsbehörde".

**Comment by jkrwdf** [comment]
Yes, this is Android and can follow the argument that this dialog is not originated in the CWA itself. 

Nevertheless, I will not close the issue yet based on the following argument: 

The CWA is for locale "DE_de" the only allowed user of the ENF-API in Android. 
The CWA project team has (certainly.. I think I heard this in the press conference to CWA release in June 2020) a direct channel to Google.

I think it could be possible to inform Google that the "Sie"-form could be used also in their dialogs.

**Comment by MikeMcC399** [comment]
@jkrwdf 

> I think it could be possible to inform Google that the "Sie"-form could be used also in their dialogs.

That is a valid suggestion, although Google uses "du" everywhere else in their dialogs, not related to COVID-19. For instance I have "Anzeigen" in the list under "COVID-19-Benachrichtigungen" and there it says: "Mit diesen Einstellungen kannst du festlegen, ob Apps die Werbe-ID dieses Geräts verwenden können, um Werbung zu personalisieren", using the du form also.

Probably this issue should be moved to the Android repository.
